docs/data_ingestion.md

```diff
@@ -41,6 +42,12 @@ The script uses the following steps to index documents:
 3. Split the PDFs into chunks of text.
 4. Upload the chunks to Azure AI Search. If using vectors (the default), also compute the embeddings and upload those alongside the text.
 
+### Enhancing search functionality with data categorization
+
+To enhance search functionality, categorize data during the ingestion process with `--category` argument, for example `scripts/prepdocs.ps1 --category ExampleCategoryName`. This argument specifies the category to which the data belongs, enabling you to filter search results based on these categories.
+
+After running the script with the desired category, ensure these categories are added to the "Include Category" dropdown list in the developer settings. The default option for this dropdown is "All". By including specific categories, you can refine your search results more effectively.
```

Comment:
Can you link to the relevant Chat/Ask files here, to make it clear what you mean by adding the categories?

Reply:
Links added, please kindly re-review.